e2e-tests/cypress/support/insurance/complete-declarations.js

```diff
@@ -1,17 +1,33 @@
 import partials from '../../e2e/partials';
+import { FIELD_VALUES } from '../../../constants';
 
 const { taskList } = partials.insurancePartials;
 
 const task = taskList.submitApplication.tasks.declarationsAndSubmit;
 
-// runs through the full declarations journey
-export default () => {
+/**
+ * completeDeclarations
+ * Runs through the full declarations journey
+ * @param {Object} Object with flags on how to complete specific declaration forms.
+ * - exportingWithCodeOfConduct: Should submit "yes" in the "exporting with code of conduct" form. Defaults to "yes".
+ */
+const completeDeclarations = ({ exportingWithCodeOfConduct = true }) => {
   task.link().click();
 
   cy.completeAndSubmitDeclarationConfidentiality();
+
   cy.completeAndSubmitDeclarationAntiBribery();
+
   cy.completeAndSubmitDeclarationAntiBriberyCodeOfConduct();
-  cy.completeAndSubmitDeclarationAntiBriberyExportingWithCodeOfConduct();
+
+  if (exportingWithCodeOfConduct === true) {
```

Comment:
could this just be if `(exportingWithCodeOfConduct) {`

Reply:
Will do this in the next PR if that's OK, so we can get this merged and deployed?